src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -120,6 +130,22 @@ static void setupSourceProjectData(@Nonnull final SphereClient sourceProjectClie
 
     sourceProjectClient.execute(TypeCreateCommand.of(typeDraft)).toCompletableFuture().join();
 
+    final TaxRateDraft draft =
```

Comment:
a small refactor for list creation:
```
final TaxRateDraft taxRateDraft =
        TaxRateDraftBuilder.of("Tax-Rate-Name-1", 0.3, false, CountryCode.DE).build();

    final TaxCategoryDraft taxCategoryDraft =
        TaxCategoryDraftBuilder.of(
                "Tax-Category-Name-1", singletonList(taxRateDraft), "Tax-Category-Name-1")
            .key(RESOURCE_KEY)
            .build();
```

Reply:
https://github.com/commercetools/commercetools-project-sync/commit/f4f354eb8092d285d2fdef52efaf669efb908efe